src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -106,7 +139,9 @@ const ChatItemMyTextV2 = ({
                   <View style={styles.chatTitleContainer}>
                     <Text style={[styles.userText, textStyle(true)]}>{username}</Text>
                     <View style={dotStyle(true)} />
-                    <Text style={[styles.timeText, textStyle(true)]}>{time}</Text>
+                    <Text testID="timestamp" style={[styles.timeText, textStyle(true)]}>
+                      {calculateTime(time, true)}
+                    </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The time is now formatted using the `calculateTime` function. This is a good improvement for readability. However, please ensure that the `calculateTime` function handles all possible time inputs correctly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->